esp8266-fastled-webserver/data/js/app.js

```diff
@@ -300,6 +302,52 @@ function addSelectField(field) {
   $("#form").append(template);
 }
 
+function addUtcOffsetIndexField(field) {
+  var template = $("#utcOffsetIndexTemplate").clone();
+
+  template.attr("id", "form-group-" + field.name);
+  template.attr("data-field-type", field.type);
+
+  var id = "input-" + field.name;
+
+  var label = template.find(".control-label");
+  label.attr("for", id);
+  label.text(field.label);
+
+  var select = template.find(".form-control");
+  select.attr("id", id);
+
+  select.val(field.value);
+
+  select.change(function () {
+    var value = template.find("#" + id + " option:selected").index();
+    postValue(field.name, value);
+  });
+
+  var previousButton = template.find(".btn-previous");
```

Comment:
Can you help me understand the benefits of prev/next buttons to switch timezones?  (e.g., over the drop-down selections that exist for palettes / patterns)

Reply:
It has both the select and prev/next buttons, just like the patterns & palettes fields.